.travis.yml

```diff
@@ -12,10 +12,8 @@ install:
 - pip install wavefront-pyformance
 - pip install flake8 flake8-colors flake8-docstrings flake8-import-order
 - pip install pep8-naming
-- pip install pylint
```

Comment:
Please bring back the `pylint` checks. Thanks!

Reply:
Will do once Span Log change of Wavefront SDK is merged, it's causing import error right now.

Reply:
BTW, we may use `pip install --index-url https://test.pypi.org/simple --extra-index-url wavefront-sdk-python` to get the latest one once it's merged.

Reply:
sure, will update once that is merged

Reply:
And instead of removing pylint, you could just move it from `before_script` section into `script` section after `unittest` if your goal is to let unit tests to run.